openedx/core/djangoapps/course_groups/urls.py

```diff
@@ -0,0 +1,53 @@
+"""
+Cohort API URLs
+"""
+
+from django.conf import settings
+from django.conf.urls import url
+
+from . import api
+
+urlpatterns = [
+    url(
+        r'^v1/{}/settings$'.format(
+            settings.COURSE_KEY_PATTERN,
+        ),
+        api.course_cohort_settings_handler,
+        name='api_cohort_settings',
+    ),
+    url(
+        r'^v1/{}/(?P<cohort_id>[0-9]+)?$'.format(
+            settings.COURSE_KEY_PATTERN,
+        ),
+        api.cohort_handler,
+        name='api_cohorts',
+    ),
+    url(
+        r'^v1/{}/(?P<cohort_id>[0-9]+)$'.format(
+            settings.COURSE_KEY_PATTERN,
+        ),
+        api.users_in_cohort,
+        name='api_list_cohort',
+    ),
+    url(
+        r'^v1/{}/(?P<cohort_id>[0-9]+)/add$'.format(
```

Comment:
Instead of an `/add` endpoint you could just PUT/POST to the above.

Reply:
:+1: